Marlin/temperature.cpp

```diff
@@ -1,19 +1,19 @@
 /*
```

Comment:
You can revert all your changes to `temperature.cpp`. I have a pending PR #1541 that deals with all these code formatting issues and does much more cleanup besides.